services/frontend/src/components/assignments/assignments-page.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Page } from '../shared';
+import { fetchAssignments } from '../../store/assignments';
+import api from '../../utils/api';
+import AssignmentsList from './assignments-list';
+import AssignmentEditor from './assignment-editor';
+import styles from './assignments-page.module.scss';
+
+export class AssignmentsPage extends React.Component {
+    constructor(props) {
+        super(props);
+        this.didSelectAssignment = this.didSelectAssignment.bind(this);
+
+        this.state = {
+            selectedAid: undefined,
+        };
+    }
+
+    componentDidMount() {
+        api.setBearerToken(
+            'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1aWQiOiJmYWJiMDQyZi05NjU2LTRjMjAtYmYzMy1hZmM5MDMzN2E1ZTEiLCJpYXQiOjE2MDE4NDM3OTB9.oaMx3ATdIOYikkdMPI4f8lnAIcS0z5hAaP6hODOQUC8'
```

Comment:
You shouldn't need to do this anymore, you should be able to just login and have a user specific bearer token

Reply:
done